docs/pages/database-access/troubleshooting.mdx

```diff
@@ -137,3 +137,18 @@ Now Alice can connect to any database in the Teleport cluster using any database
 This example is intentionally simple; we could have configured Alice's permissions using more fine-grained control.
 For more detailed information about database access controls and how to restrict
 access see the [RBAC](../database-access/rbac.mdx) documentation.
+
+## Connection to MySQL database results in "Unknown system variable 'query_cache_size'" error
+When a Teleport Proxy is configured with Teleport Proxy Listener Mode (`proxy_listener_mode: separate`) by default MySQL client connections to the Teleport Proxy don't use the TLS Routing feature that allows propagating correct MySQL Server Version.
```

Comment:
I think the structure of this sentence pretty ambiguous. Does this mean that TLS Routing allows propagating the correct MySQL server version, or that, by not using TLS Routing, the Proxy Service can propagate the correct version?

Reply:
@ptgott 
Yes, With TLS Routing enabled the MySQL Engine version propagation works correctly and this  issue occurs only for setups where TLS Routing is disabled. 

I have rephrased this sentence to: 
> When TLS Routing feature is disable by default, the Teleport Proxy Service returns `8.0.0-Teleport` as the MySQL server version. In some cases, like connecting with a GUI Client, this can result in obtaining an `Unknown system variable 'query_cache_size'` error that indicates that MySQL capabilities were not properly negotiated between the MySQL client and server.